docs/scraping-service.md

```diff
@@ -0,0 +1,125 @@
+# Scraping Service Mode
+
+Scraping Service Mode is a third operational mode of the Grafana Cloud Agent
+that allows for clustering a set of Agent processes and distributing scrape load
+between them.
+
+Determining what to scrape is done by storing instance configuration files to an
+[API](./api.md) which then stores the configuration files in a KV store backend.
+All agents in the cluster must use the same KV store so they read the same set
+of config files.
+
+The "instance" refers to the process of:
+
+- Service discovery for all `scrape_configs` within that loaded config
+- Scrapes metrics from all discovered targets
+- Stores data in its own Write-Ahead Log specific to the loaded config
+- Remote Writes scraped metrics to the configured `remote_write` clients
+  specified within the loaded config.
+
+
+The "instance configuration file," then, is the configuration file that
```

Comment:
ah short example of what (or why you would use) multiple "instance" configs might look like may be useful